### DISCUSSION TITLE
Sort by Session Name in Favourites

### DISCUSSION THREAD

**Discussion by GSM** [rustdesk/rustdesk#5338] [Feature Request]
_Sorting by ID_ is pretty much useless, the same goes for _Remote Host_ and _Username_ - usually RD's users cannot influence them, although there might be legit scenarios for sorting by those properties.
Much more helpful would be sorting by session's name, since one can rename them or even come up with some naming scheme.

**Comment by FNW2022** [comment]
I support this already rather old improvement request as I would like to be able to sort my _Favorites_ by the "_Defined name_" I did explicitly set for each entry:
![image](https://github.com/user-attachments/assets/781ce820-27a6-496b-b7c9-733d971ae877)
So after using the Rename function on each entry, they all bear a "**defined name**" (as I call it to distinguish it from the _Username_ which reflects the name of the OS user on the other end of the line).
This has the most meaning to me as I can choose it freely and I am not bound to anything coming from the client computer...

It would be great if the _Sorting_ option could be extended by a fifth option (however you name it) to allow sorting the list of _Favorites_ (or any other list) by the self-defined name of each entry (as proposed in my screenshot above).
